extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -468,7 +467,7 @@ export class AmpA4A extends AMP.BaseElement {
       'au': adQueryIdx < 0 ? '' :
           this.adUrl_.substring(adQueryIdx + 1, adQueryIdx + 251),
     };
-    return new Error('amp-a4a: ' + JSON.stringify(state));
+    throw new Error('amp-a4a: ' + JSON.stringify(state));
```

Comment:
agree that the current code needs to be improved. 

but line 453 still returns an error object though. do you want to fix that as well?